django_cloud_deploy/tests/unit/skeleton/requirements_parser_test.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit test for django_cloud_deploy/skeleton/requirements_parser.py."""
```

Comment:
```suggestion

"""Unit test for django_cloud_deploy/skeleton/requirements_parser.py."""
```

Reply:
Somehow yapf is not happy with this: [Link](https://circleci.com/gh/GoogleCloudPlatform/django-cloud-deploy/2400?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)